CHANGES.md

```diff
@@ -60,7 +60,7 @@
 
 ## Breaking Changes
 
-# [2.23.0] - Unreleased
```

Comment:
Don't forget to put the date here once the release is finalized

Reply:
Good catch, thanks!